test/test_sparse_csr.py

```diff
@@ -955,6 +957,23 @@ def test_coo_csr_conversion(self, device, dtype):
 
             self.assertEqual(csr_sparse.to_dense(), dense)
 
+    @ops(_sparse_csr_ops)
+    def test_sparse_csr_no_error(self, device, dtype, op):
+        samples = op.sample_inputs(device, dtype)
+
+        if len(samples) == 0:
+            self.skipTest("Skipped! No sample inputs!")
+
+        for sample in samples:
+            assert isinstance(sample.input, torch.Tensor)
+            if sample.input.ndim != 2:
+                continue
+
+            expected = op(sample.input)
+            assert torch.is_tensor(expected)
+            output = op(sample.input.to_sparse_csr())
+            assert torch.is_tensor(output)
```

Comment:
The plan is to look at it again, and have a specialized test for sparse CSR tensors ((if everyone agrees)).

I plan to take a look at it in a separate PR next week.